style.css

```diff
@@ -0,0 +1,693 @@
+
+.toggle-menu {
+  position: absolute;
+  display: none;
+  width: 100%;
+  height: 112vh;
+  top: 0;
+  background: #fff;
+  color: #333;
+  z-index: 10;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
+
+.menu {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.active-border {
+  padding: 5px 12px;
+  border: 5px solid #ec5242;
+}
+
+.menu-link {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.menu-link:hover {
+  color: #ec5242;
+}
+
+.menu-list {
+  margin: 20px 0;
+}
+
+.hambug-items {
+  z-index: 12;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  top: 100px;
+}
+
+#togclose {
+  position: absolute;
+  right: 50px;
+  top: 50px;
+  cursor: pointer;
+}
+
+.hambuger {
+  border-radius: 10px;
+  width: 18px;
+  margin-top: 10px;
+  margin-right: 30px;
+  margin-bottom: 0;
+  cursor: pointer;
+}
+
+.speakers {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 15px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.card img {
+  width: 100%;
+  margin-right: 30px;
+  flex-shrink: 0;
+}
+
+.about-speakers {
+  margin: 10px 20px;
+  width: 60%;
+}
+
+.about-speakers h3 {
+  margin-bottom: 10px;
+}
+
+.about-speakers h4 {
+  margin-bottom: 10px;
+  color: #ec5242;
+}
+
+.about-speakers p {
+  font-size: 16px;
+}
+
+.first-section {
+  height: 105vh;
+  background-image: url(images/background.jpg);
+  background-size: cover;
+}
+
+.first-content {
+  display: flex;
+  justify-content: center;
+  text-align: left;
+  flex-direction: column;
+  height: 50vh;
+}
+
+.transparent {
+  height: 105vh;
+  background-color: #fff;
+  opacity: 80%;
+}
+
+.text {
+  -webkit-background-clip: text;
+  background-image: linear-gradient(90deg, #fd1900, #f06657, #333);
+  color: transparent;
+  font-weight: bold;
+  font-size: 31px;
+  font-family: goose;
+  margin: 0 15px;
+}
+
+.text-hello {
+  color: #ec5242;
+  font-weight: bold;
+  font-size: 31px;
+  font-family: "Lato", sans-serif;
+  margin: 0 15px;
+}
+
+.text2 {
+  font-weight: bold;
+  font-size: 18px;
+  text-align: center;
+}
+
+.text3 {
+  color: #272a31;
+  font-weight: bold;
+  font-size: 25px;
+  margin-left: 17px;
+  font-family: goose;
+}
+
+.describe {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 25vh;
+}
+
+.describe2 {
+  margin-top: 5px;
+  display: flex;
+  align-items: center;
+  height: 25vh;
+  justify-content: center;
+}
+
+.second-section {
+  height: auto;
+  background-color: #272a31;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.text4 {
+  color: #fff;
+}
+
+.line {
+  position: relative;
+  left: 40px;
+  border-top: #ec5242 2px solid;
+  width: 60px;
+}
+
+.flexbox {
+  margin-top: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
```

Comment:
- [ ] I can see that you have an issue with the width of the Main Program section container :point_down: 

![111](https://user-images.githubusercontent.com/57562869/148447650-5272db17-635d-4d53-bd4f-f404fb1cf0f5.png)

- I think you miss a `width: ` property, kindly add width to this container to solve the issue eg (`width: 90%`)